packages/test-helpers/package.js

```diff
@@ -24,13 +24,25 @@ Package.onUse(function (api) {
 
 
   api.export([
-    'pollUntil', 'try_all_permutations',
-    'SeededRandom', 'clickElement', 'blurElement',
-    'focusElement', 'simulateEvent', 'getStyleProperty', 'canonicalizeHtml',
-    'renderToDiv', 'clickIt',
-    'withCallbackLogger', 'testAsyncMulti',
-    'simplePoll', 'runAndThrowIfNeeded',
-    'makeTestConnection', 'DomUtils']);
+    'blurElement',
+    'canonicalizeHtml',
+    'clickElement',
+    'clickIt',
+    'DomUtils',
+    'focusElement',
+    'getStyleProperty',
+    'makeTestConnection',
+    'MockFunction',
+    'pollUntil',
+    'renderToDiv',
+    'runAndThrowIfNeeded',
+    'SeededRandom',
+    'simplePoll',
+    'simulateEvent',
+    'testAsyncMulti',
+    'try_all_permutations',
+    'withCallbackLogger',
+  ]);
```

Comment:
I completely understand if you want me to revert this and just add the single thing to make this diff better. I just ... really wanted to make this cleaner for the future.

Reply:
I'm fine with this.